chemist/src/main/scala/flask.scala

```diff
@@ -76,21 +99,16 @@ object Flask {
    * Call out to the specific location and grab the list of things the flask
    * is already mirroring.
    */
-  private def requestAssignedTargets(location: Location)(http: dispatch.Http): Task[Set[Target]] = {
+  private def requestAssignedTargets(location: Location)(http: dispatch.HttpExecutor): Task[Set[Target]] = {
     import argonaut._, Argonaut._, JSON._, HJSON._
     import dispatch._, Defaults._
 
     val a = location.uriFromTemplate(LocationTemplate(s"http://@host:@port/mirror/sources"))
     val req = Task.delay(url(a.toString)) <* Task.delay(log.debug(s"requesting assigned targets from $a"))
-    req flatMap { b =>
-      http(b OK as.String).map { c =>
-        Parse.decodeOption[List[Cluster]](c
-        ).toList.flatMap(identity
-        ).map { cluster =>
-          log.debug(s"Received cluster $cluster from $a")
-          cluster
-        }.foldLeft(Set.empty[Target]){ (a,b) =>
-          a ++ b.urls.map(s => Target(b.label, new URI(s))).toSet
+    req flatMap {
```

Comment:
Can we adjust formating to similar to how it was before? The new version is a little hard to read IMHO (granted, this is a complicated but key bit of code).